Makefile

```diff
@@ -170,6 +170,10 @@ index-cleaner-integration-test: docker-images-elastic
 index-rollover-integration-test: docker-images-elastic
 	$(MAKE) storage-integration-test COVEROUT=cover-index-rollover.out
 
+.PHONY: tail-sampling-integration-test
+tail-sampling-integration-test:
+	SAMPLING=tail $(MAKE) jaeger-v2-storage-integration-test
```

Comment:
this runs `go test`, but when do you start the docker compose environment?

All other e2e tests have a driver script that orchestrates all components of the test, e.g. `scripts/es-integration-test.sh`

Reply:
I'm not using the docker-compose environment for my test. Calling `e2eInitialize` is enough to start the Jaeger collector. You can simply run this test by calling `make tail-sampling-integration-test`. Let me know if you want to change any of this setup though.

Reply:
Fair enough. However, it means that the new docker compose file will begin to rot since it's not being exercised by the CI, something we tried to avoid (e.g. see e2e spm test). So it would be good to actually combine using docker compose with e2e test.

Reply:
Ah I see! What would you want this to look like? The current docker-compose set up generates load using tracegen which ideally we wouldn't want in the integration test so we can manually generate those.

Reply:
Well, that's why I was asking from the start what your plan would be. The way you are using e2e_integration framework is very lightweight, and I could easily see an alternative setup where everything is just orchestrated from a shell script
  * run docker-compose with one config
    * maybe don't include tracegen in compose, run it manually 
  * do a curl against query service to retrieve service names as JSON (trivial to write)
  * shut down docker-compose (to clear the storage) and run again with different config

If you are interested to pursue this approach, I would suggest still merging this PR first so that we already have _something_ in place. Can you finish the README?

Reply:
@yurishkuro That sounds good to me and I can pursue that approach in a follow-up PR. And yes, working on the README and changelog now. Will push it up soon.